packages/react/src/components/Modal/Modal.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta, Primary, Controls, Canvas, ArgTypes } from '@storybook/blocks';
+import * as ModalStories from './Modal.stories';
+
+import { Modal } from './';
+
+<Meta of={ModalStories} />
+
+# Modal
+
+`Modal` bruker native `dialog` elementet, og for å åpne/lukke må du bruke `ref`.
+Dette er valgt for å ha en så lett modal som mulig, og den vil derfor fungere annerledes enn våre andre komponenter.
+Les [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) for mer info.
+
+<Primary />
+<Controls />
+
+Legg merke til at modalen har focus trap, som betyr at brukeren ikke kan tabbe til annet innhold på siden.
+
+## Close on backdrop click
+
+<Canvas of={ModalStories.CloseOnBackdropClick} />
+
+## Form
+
+Vi bruker native `autoFocus` på `Textfield` for å fokusere inputen i skjemaet.
+
+<Canvas of={ModalStories.ModalWithForm} />
+
+### `Modal.Header`
+
+<ArgTypes of={Modal.Header} />
+
+### `Modal.Footer`
+
+<ArgTypes of={Modal.Footer} />
```

Comment:
Suggest removing this as it has no props. 

![image](https://github.com/digdir/designsystem/assets/1070981/194b1545-d45e-4d9e-a811-98b5d2d482b2)